src/components/SideBar/SideBar.jsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import membersImg from './style/members.png';
+import tasksImg from './style/tasks.png';
+import taskTracksImg from './style/taskTracks.png';
+
+const sideBaerList = [
+  { name: 'Members', path: '/', img: membersImg },
+  { name: 'Tasks', path: '/tasks', img: tasksImg },
+  { name: 'Task tracks', path: '/task-tracks', img: taskTracksImg },
+];
+
+export const SideBar = ({ activeSideBarElem, onClick }) => (
+  <div className='sideBar'>
+    <div className='sideBar__container'>
+      {sideBaerList.map((item, index) => (
```

Comment:
`...sideBarList`

Reply:
thx for attentiveness, it's so ashamed for me)
```suggestion
      {sideBarList.map((item, index) => (
```